### ISSUE TITLE
Uncaught TypeError: atom.workspace.getLeftPanels(...)[0].getItem(...).selectedPaths is not a func...

### ISSUE THREAD

**Issue by Bulletninja** [dingjie/atom-sync#14]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.19
**System**: Mac OS X 10.11.1
**Thrown From**: [atom-sync](https://github.com/dingjie/atom-sync) package, v0.5.2

### Stack Trace

Uncaught TypeError: atom.workspace.getLeftPanels(...)[0].getItem(...).selectedPaths is not a function

```
At /Users/luis/.atom/packages/atom-sync/lib/atom-sync.coffee:18

TypeError: atom.workspace.getLeftPanels(...)[0].getItem(...).selectedPaths is not a function
    at atom-workspace.subscriptions.add.atom.commands.add.atom-sync:upload-directory (/Users/luis/.atom/packages/atom-sync/lib/atom-sync.coffee:18:76)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:145:19)
    at [object Object].jQuery.fn.trigger (/Applications/Atom.app/Contents/Resources/app.asar/src/space-pen-extensions.js:64:23)
    at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:75:67)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
    at EventEmitter.ipc.sendSync (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/ipc.js:21:31)
    at BrowserWindow.RemoteMemberFunction [as emit] (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/remote.js:140:29)
```

### Commands

```
     -3:00.3.0 application:new-file (atom-pane.pane.active)
     -2:59.5.0 application:open-file (atom-text-editor.editor)
     -0:24 core:paste (atom-text-editor.editor.is-focused)
     -0:22.8.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "disabledPackages": [
      "line-count-status"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v0.2.4
atom-material-ui, v0.6.2
atom-sync, v0.5.2
tool-bar, v0.1.9
tool-bar-main, v0.0.8

# Dev
No dev packages
```

**Comment by Bulletninja** [comment]
Seems like a problem with tool-bar, if i hide it, atom-sync works again.

**Comment by dingjie** [comment]
Should be resolved in [0.6.0](https://github.com/dingjie/atom-sync/releases/tag/v0.6.0), which does not determine selected item from panel
